lean4-infoview-api/src/infoviewApi.ts

```diff
@@ -128,3 +129,9 @@ export interface InfoviewApi {
    */
   requestedAction(action: InfoviewAction): Promise<void>;
 }
+
+export interface TestApi {
+  isInfoViewOpen() : Promise<boolean>;
+  copyHtmlToClipboard(): Promise<boolean>;
```

Comment:
```suggestion
  getHtmlContents(): Promise<string>;
```
I think that this is the operation that we actually want, and then we don't need to do the awkward `sleep(500)` in the test.

Reply:
Sure, this is cleaner.  Note this just pushes the retry loop lower, see https://github.com/lovettchris/vscode-lean4/blob/clovett/unittest/vscode-lean4/src/infoview.ts#L346

Reply:
Wojciech showed me how to make this synchronous so no retry loop necessary.